packages/app/src/cli/utilities/developer-platform-client/shopify-developers-client.ts

```diff
@@ -258,6 +344,10 @@ export class ShopifyDevelopersClient implements DeveloperPlatformClient {
   async appPreviewMode(_input: FindAppPreviewModeVariables): Promise<FindAppPreviewModeSchema> {
     throw new BugError('Not implemented: appPreviewMode')
   }
+
+  toExtensionGraphQLType(input: string) {
+    return input.toLowerCase()
```

Comment:
Partners returns uppercase, while Shopify Developers returns lowercase (at least for now).